documentation/staging/content/faq/external-clients.md

```diff
@@ -366,6 +381,129 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
     * For operator hosted WebLogic Servers, you can set this property by including `-Dweblogic.rjvm.allowUnknownHost=true` in the `JAVA_OPTIONS` [Domain environment variable]({{< relref "/userguide/managing-domains/domain-resource#jvm-memory-and-java-option-environment-variables" >}}) defined in the domain resource's `spec.serverPod.env` attribute.
   * Also apply patch 30656708 on each target WebLogic Server for versions 12.2.1.4 (PS4) or earlier.
 
+#### Configuring WebLogic Server Affinity Load Balancing Algorithm
+
+When providing external clients access to EJB and JMS resources using load balancer tunneling, it is recommended to configure a server affinity load balancing algorithm for the cluster in which the resources are targeted.  Using a server affinity based algorithm turns off load balancing between external Java clients and server instances causing method calls to 'stick' to a server instance to which the external client is communicating with.  For more information see [Load Balancing for EJBs and RMI Objects](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/clust/load_balancing.html#GUID-2470EEE9-F6F9-44EF-BA54-671728E93DE6) in _Load Balancing in a Cluster_.
```

Comment:
IMO this doesn't really 'turn off load balancing' from a customer perspective, since RMI stub stub load balancing is a somewhat hidden feature, and there was already load balancing decision made for the JNDI context itself when the JNDI context was created.

How about something like:

Using a server affinity based algorithm reduces the amount of time it takes for EJB and JMS stand-alone or server hosted clients to establish a connection through a tunneling port.  It does this by ensuring that the clients prefer communicating with a target server instance which already has an established connection to the client, where this server instance is implicitly picked when such clients create their JNDI context.